instance/config.py

```diff
@@ -3,15 +3,15 @@
 class Config(object):
     #parent configuration class
     DEBUG = False
-    CSRF_ENABLED = True
+    #CSRF_ENABLED = True
```

Comment:
Do you need this?

Reply:
No. Should I remove them?

Reply:
Yes clean up commented out code that you are not using